roles/nfs_volumes/library/partitionpool.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+"""
+Ansible module for partitioning.
```

Comment:
@markturansky understood, my question was more specifically about the partitionpool ansible module specifically and not about the nfs-volumes role itself. At first glance it looked relatively generic but after taking a closer look, I agree with @jsafrane that it would probably not be accepted upstream without more generic partitioning support.